project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.1.6
+sbt.version=1.1.4
```

Comment:
We went back?

Reply:
weird. in any case the sbt updates seems to alternatively break and fix triggered compilation

Reply:
My editor integration seems to work better with 1.1.4 but I'll put it back before merging.

Reply:
Perhaps we should keep 1.1.4. Triggered execution is being very slow on 1.1.6 though in theory it was fixed yet again on 1.1.5 and 1.1.6

Reply:
Fine with me. I'll remove that todo item.